OOP Fundamentals and C#/09 - CreationalPatterns/Factory/FactoryMethod/ITransportFactory.cs

```diff
@@ -3,5 +3,7 @@
     public interface ITransportFactory
```

Comment:
Could you use please abstract class instead of interface and hold logic of "operation" method here